browser-extension/mv3/rollup.config.js

```diff
@@ -101,9 +101,17 @@ export default [
     plugins: commonPlugins,
   },
   {
-    input: "src/client-scripts/index.ts",
+    input: "src/client-scripts/sessionRecorderHelper.js",
     output: {
-      file: `${OUTPUT_DIR}/client.js`,
+      file: `${OUTPUT_DIR}/client-scripts/sessionRecorderHelper.js`,
```

Comment:
Should we rename the folder to `page-scripts` and rename the files output to `sessionRecorderHelper.ps.js` and `ajaxRequestInterceptor.ps.js`?

I get confused a lot between content-script and client-script as there is another client folder with content-scripts